### ISSUE TITLE
MAC will not be saved to config file

### ISSUE THREAD

**Issue by Thor77** [Thor77/Blueproximity#7]
Hi,...

in 1.1.7 the MAC of the preferred device will not be saved to config file.

i added the MAC by hand to "~/.blueproximityrc" and i works wonderfull.

Thank you for this very nice piece of software!

i'm using:

Fedora Core 6 (2.6.22.2-42.fc6)

with:

gnome-python2-2.16.2-2.fc6
pybluez-0.9.1-3.fc6
python-configobj-4.3.2-5.fc6
bluez-libs-3.9-1.fc7
gnome-bluetooth-libs-0.7.0-12.fc6
bluez-utils-3.9-1.fc7
bluez-gnome-0.6-2.fc7
bluez-hcidump-1.33-1.fc7
gnome-bluetooth-0.7.0-12.fc6
gnome-bluetooth-devel-0.7.0-12.fc6
bluez-libs-devel-3.9-1.fc7

Bye
--

**Comment by Thor77** [comment]
Logged In: YES 
user_id=114903
Originator: NO

I consider this one to be closed then.
(By highno)

**Comment by Thor77** [comment]
Logged In: YES 
user_id=114903
Originator: NO

I see that I used the wrong signals in the .glade file. I changed them to something useful in SVN Version and will release a 1.1.8 tonight which includes these changes.
I still have to find a better signal for the Combobox-Entries found on the third tab. They fire at every single change which then writes the configuration every time you type a single character. Still it is usable and works, though this has to be changed in future versions to a more i/o friendly way.

Bye,
Lars
(By highno)
